src/main/java/com/reshiftsecurity/plugins/intellij/actions/HelpAction.java

```diff
@@ -120,7 +120,7 @@ private static StringBuilder createHelpInfo() {
 		ret.append(String.format("Support & Feedback: <a href='mailto:%s'>dev@reshiftsecurity.com</a>", PluginConstants.RESHIFT_DEV_EMAIL));
 		ret.append("<br/>");
 		ret.append("<br/>");
-		ret.append("Would you like to integrate Reshift into your CI pipline? Sign up today for free on <a href='")
+		ret.append("Would you like to integrate Reshift into your CI pipeline? Sign up today for free on <a href='")
```

Comment:
is there a tool to spell check our string literals?

Reply:
Yes, will look into integrating something in our CD pipeline